client/components/_tests_/footer.test.tsx

```diff
@@ -0,0 +1,11 @@
+import Footer from "../footer";
+import { render } from '@testing-library/react';
+
+describe("footer test", () => {
+    it('is redendring properly footer component ', () => {
+        const { getByTestId } = render(<Footer />);
+        const elementId = getByTestId('footer');
```

Comment:
you should check if the links id are inside of footer, and not footer.

Reply:
what link ? footer has no link inside of it, layout has a link of footer, when you said id i was thinking to check if footer is creating the component footer and the same for header